### PR TITLE
SWTCH-1305 rename properties

### PR DESCRIPTION
**Pull request by dawidgil**

### CODE REVIEW

File: src/app/routes/applications/governance-view/governance-details/governance-details.component.ts [hunk 1]
```diff
@@ -5,9 +5,9 @@ import { NamespaceType, PreconditionType } from 'iam-client-lib';
 import { ListType } from '../../../../shared/constants/shared-constants';
 import { IamService } from '../../../../shared/services/iam.service';
 import { LoadingService } from '../../../../shared/services/loading.service';
-import { RoleType } from '../../new-role/new-role.component';
 import { GovernanceViewComponent } from '../governance-view.component';
 import { IssuerType } from '../../new-role/models/issuer-type.enum';
+import { IFieldDefinition } from '@energyweb/iam-contracts/dist/src/types/DomainDefinitions';
```
Comment (Harasz) [inline src/app/routes/applications/governance-view/governance-details/governance-details.component.ts:10]: Isn't it directly export from `@energyweb/iam-contracts`?

File: src/app/routes/enrolment/view-requests/view-requests.component.ts [hunk 2]
```diff
@@ -15,6 +15,8 @@ import { Store } from '@ngrx/store';
 import { UserClaimState } from '../../../state/user-claim/user.reducer';
 import * as userSelectors from '../../../state/user-claim/user.selectors';
 import { EnrolmentForm } from '../../registration/enrolment-form/enrolment-form.component';
+import { IFieldDefinition } from '@energyweb/iam-contracts/dist/src/types/DomainDefinitions';
```
Comment (Harasz) [inline src/app/routes/enrolment/view-requests/view-requests.component.ts:18]: :negative_squared_cross_mark:

File: src/app/routes/enrolment/view-requests/view-requests.component.ts [hunk 3]
```diff
@@ -167,13 +169,26 @@ export class ViewRequestsComponent implements OnInit {
...
-        this.fields = decoded.claimData?.fields
-          ? decoded.claimData?.fields
-          : [];
+        this.requestorFields = this.getRequestorFields(decoded.claimData);
       }
     }
   }
 
+  private getRequestorFields(data: {
+    requestorFields?: KeyValue<string, string | number>[];
```
Comment (Harasz) [inline src/app/routes/enrolment/view-requests/view-requests.component.ts:178]: Maybe we can extract this this, ex `type FieldsKeyValue = KeyValue<string, string | number>`
```suggestion
    requestorFields?: FieldsKeyValue[];
```
I saw that we using it in couple of places

Comment (dawidgil) [reply src/app/routes/enrolment/view-requests/view-requests.component.ts:178]: Good idea

File: src/app/routes/applications/governance-view/governance-details/governance-details.component.ts (hunk 1 above)
Comment (dawidgil) [reply src/app/routes/applications/governance-view/governance-details/governance-details.component.ts:10]: It's not reexported :/ So i need to have import like this. interface in general is exported, but in index.ts is not imported and later on reexported. The best would be just export everything imo.
image

Comment (Harasz) [reply src/app/routes/applications/governance-view/governance-details/governance-details.component.ts:10]: https://github.com/energywebfoundation/iam-contracts/pull/111

File: src/app/routes/enrolment/view-requests/view-requests.component.ts (hunk 3 above)
Comment (Harasz) [reply src/app/routes/enrolment/view-requests/view-requests.component.ts:178]: Do you want to change it in this PR or later?

Comment (dawidgil) [reply src/app/routes/enrolment/view-requests/view-requests.component.ts:178]: is iam-client-lib adjusted with those changes? If yes, then i will do it in this PR

**Review by Harasz — APPROVED** [review]

**Pull request merged**